src/main/kotlin/no/nav/klage/pdfgen/service/PDFGenService.kt

```diff
@@ -44,10 +44,14 @@ class PDFGenService {
         return os.toByteArray()
     }
 
-    private fun getHTMLDocument(list: List<Map<String, *>>): Document {
+    fun validateDocumentContent(json: String) {
+        getHTMLDocument(jacksonObjectMapper().readValue(json, List::class.java) as List<Map<String, *>>, true)
+    }
+
+    private fun getHTMLDocument(list: List<Map<String, *>>, validationMode: Boolean = false): Document {
         validateHeaderFooter(list)
         val c = HtmlCreator(list)
-        return c.getDoc()
+        return c.getDoc(validationMode)
```

Comment:
Jeg lurer på om `validationMode` kunne vært en (global) property på `HtmlCreator`-klassen i stedet for å bli sendt rundt i diverse metoder?

Reply:
Ikke dumt! Skal gjøre et slik forsøk.